### PR TITLE
feat(EMS-2754): No PDF - Policy - Loss payee - Page setup

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
This PR sets up a new page/form for a "Loss payee" in the policy section of a no PDF application.

## Resolution :heavy_check_mark:
- Create route, constants, field and page content strings.
- Add E2E test.
- Create new GET controller and simple POST controller/endpoint.
- Update "Confirm broker address" and "Using broker" POST to redirect to "Loss payee", instead of "Check your answers".

## Miscellaneous :heavy_plus_sign:
- Minor unit test improvements.

### CODE REVIEW

**Review by Zainzzkk — DISMISSED** [review]

File: e2e-tests/commands/insurance/complete-policy-section.js [hunk 1]
```diff
@@ -55,6 +55,8 @@ const completePolicySection = ({
     cy.clickSubmitButton();
   }
 
+  cy.completeAndLossPayeeForm();
```
Comment (Zainzzkk) [inline e2e-tests/commands/insurance/complete-policy-section.js:58]: should this be complete and submit?

File: e2e-tests/insurance/cypress/e2e/journeys/policy/different-name-on-policy/changing-from-different-name-to-same-name-then-back-to-different-name.spec.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ context(`Insurance - Policy - Different name on Policy page - Changing ${OTHER_N
       cy.completeAndSubmitPreCreditPeriodForm({});
       cy.completeAndSubmitAnotherCompanyForm({});
       cy.completeAndSubmitBrokerForm({});
+      cy.completeAndLossPayeeForm();
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/policy/different-name-on-policy/changing-from-different-name-to-same-name-then-back-to-different-name.spec.js:39]: should this be completeAndSubmit?

File: e2e-tests/insurance/cypress/e2e/journeys/policy/different-name-on-policy/changing-from-same-name-to-different-name.spec.js [hunk 3]
```diff
@@ -35,6 +35,7 @@ context(`Insurance - Policy - Different name on Policy page - Changing ${SAME_NA
       cy.completeAndSubmitPreCreditPeriodForm({});
       cy.completeAndSubmitAnotherCompanyForm({});
       cy.completeAndSubmitBrokerForm({});
+      cy.completeAndLossPayeeForm();
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/policy/different-name-on-policy/changing-from-same-name-to-different-name.spec.js:38]: same as above

Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/policy/different-name-on-policy/changing-from-same-name-to-different-name.spec.js:38]: Whoops, fixed

**Review by Zainzzkk — DISMISSED** [review]

**Pull request merged**